A note for CLion users, do not get confused (like I did) by output like this in the CMake window:
```
Problems were encountered while collecting compiler information:
	C:\Users\<xxx>\AppData\Local\Temp\compiler-file7019753171835842146: fatal error C1083: Cannot open include file: 'pxr/usd/_ar/pypch.h': No such file or directory
```

This is an artifact of the CLion "intellisense" and not related to the USD build system...

--- a/cmake/defaults/msvcdefaults.cmake
+++ b/cmake/defaults/msvcdefaults.cmake
@@ -134,3 +134,9 @@ set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /Gm-")
 # with no symbols in it.  We do this a lot because of a pattern of having
 # a C++ source file for many header-only facilities, e.g. tf/bitUtils.cpp.
 set(CMAKE_STATIC_LINKER_FLAGS "${CMAKE_STATIC_LINKER_FLAGS} /IGNORE:4221")
+
+# Enforce synchronous PDB writes when using Ninja
+# (this prevents "permission denied" compile errors on program databases)
+if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
+    set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /FS")
+endif()

